src/app/shared/components/template/components/text-bubble/text-bubble.component.html

```diff
@@ -1,21 +1,28 @@
 <div
-  class="container"
+  class="text-bubble-container container"
   [attr.data-position]="params.speakerPosition"
   [attr.data-variant]="params.variant"
 >
-  <div class="text-bubble" [attr.data-position]="params.speakerPosition">
-    @if (_row.value) {
-      <plh-tmpl-text
-        [row]="{ _nested_name: '', name: '', type: 'text', value: value() }"
-      ></plh-tmpl-text>
-    }
-    @for (childRow of _row.rows | filterDisplayComponent; track trackByRow($index, childRow)) {
-      <plh-template-component
-        [row]="childRow"
-        [parent]="parent"
-        [attr.data-rowname]="_row.name"
-      ></plh-template-component>
+  <div>
+    @if (params.speakerName) {
```

Comment:
In order to preserve consistency with non-Kuwait themes, we might need to make this element hidden on every other theme and only display it when the `plh_kids_kw` is active. Otherwise, we should style it so that it looks good on the other themes too. It would also be good to add an example to the [comp_text_bubble](https://docs.google.com/spreadsheets/d/1Yo8X4TrTW2lzEgusDplAQ-y-JJvQOIqXGNLVON1uvRo/edit?gid=569531329#gid=569531329) template so that we can see how it displays